dev/tests/integration/testsuite/Magento/Framework/View/Element/TemplateTest.php

```diff
@@ -116,6 +116,7 @@ public function testGetObjectData()
 
     public function testGetCacheKeyInfo()
     {
+        $this->_block->setTemplate($this->_block->getTemplate() ?? '');
```

Comment:
Could we try to initialize block like 
`Bootstrap::getObjectManager()->get(LayoutInterface::class)->getBlock(TopmenuBlock::class);`
(in all tests)

Reply:
Hello @xmav 
We tried to fix it using `Bootstrap::getObjectManager()->get(LayoutInterface::class)->createBlock`
Also, we rechecked it using `Bootstrap::getObjectManager()->get(LayoutInterface::class)->getBlock` 
It doesn't work for these tests.